test/integration/controller/jobs/job/job_controller_test.go

```diff
@@ -265,13 +265,22 @@ var _ = ginkgo.Describe("Job controller", ginkgo.Ordered, ginkgo.ContinueOnFailu
 		}, util.ConsistentDuration, util.Interval).Should(gomega.Succeed())
 
 		ginkgo.By("checking the workload is finished when job is completed")
+		createdJob.Status.StartTime = ptr.To(metav1.Now())
```

Comment:
is there any chance of flakiness if metav1.Now() returns a later time in one of the following calls? if so, can we store as variable and reuse it? otherwise, i like it this way

Reply:
I think that there is no flakiness since we verify only .type and .status in Line 287.

Reply:
I can't see some validation on Kubernetes part. But to be make sure that it will not happens I added now variable to use common time and using reconciler ([2d77757](https://github.com/kubernetes-sigs/kueue/pull/4029/commits/2d777578e01f9c1ce1134da9e971aff42e750264)).